src/ui/public/test_harness/test_sharding/get_shard_num.js

```diff
@@ -0,0 +1,28 @@
+import murmurHash3 from 'murmurhash3js';
+
+// murmur hashes are 32bit unsigned integers
+const MAX_HASH = Math.pow(2, 32);
+
+/**
+*  Determine the shard number for a suite by hashing
+*  it's name and placing it based on the hash
+*
+*  @param {number} shardTotal - the total number of shards
+*  @param {string} suiteName - the suite name to hash
+*  @return {number} shardNum - 1-based shard number
+*/
+export function getShardNum(shardTotal, suiteName) {
```

Comment:
The fact that their values share context ("shard" in this case) is important to naming, but it's equally important for the names to reflect distinctions. In this case, the "shard_num" identifies the shard we're executing code in.  But `getShardNum` isn't getting the `shard_num` (which is literally what the name communicates)... it's getting the shard in which a particular suite should execute.

I think we can convey the relationship more clearly with a name that reflects the shared context _and_ the differences: `getShardForSuite`, `getShardNumForSuite`, `getSuiteShard`, etc.